src/MultiRegion/cubed_sphere_boundary_conditions.jl

```diff
@@ -114,12 +114,10 @@ function fill_cubed_sphere_halo_regions!(field::CubedSphereField{<:Face, <:Face}
     return nothing
 end
 
-fill_cubed_sphere_halo_regions!(fields::Tuple{CubedSphereField, CubedSphereField};
-                                signed = true) = fill_cubed_sphere_halo_regions!(fields...; signed)
+fill_halo_regions!(fields::Tuple{CubedSphereField, CubedSphereField}; signed = true) = fill_halo_regions!(fields...; signed)
```

Comment:
is it clean/good idea to extend `fill_halo_regions!` for (::Tuple{Fields})? I guess we need something like that when dealing with vector-like quantities

cc @glwagner, @siddharthabishnu, @simone-silvestri

Reply:
there is already a method for filling the halos of tuples (and namedtuples) of fields. In that case, we launch kernels that fill all the fields together.
See 
https://github.com/CliMA/Oceananigans.jl/blob/f7042b738da746cbb6ba752e01c3894b09b6f1a4/src/Fields/field_tuples.jl#L56
and
https://github.com/CliMA/Oceananigans.jl/blob/f7042b738da746cbb6ba752e01c3894b09b6f1a4/src/BoundaryConditions/fill_halo_regions.jl#L45-L48

Reply:
In case of a `MultiRegionField` (of which a CubedSphereField` is a subset) this behavior is bypassed
https://github.com/CliMA/Oceananigans.jl/blob/f7042b738da746cbb6ba752e01c3894b09b6f1a4/src/MultiRegion/multi_region_boundary_conditions.jl#L34-L36